src/GraphQL/GlobalSwitches.cs

```diff
@@ -54,5 +55,23 @@ public static class GlobalSwitches
         /// member will be unusable.
         /// </summary>
         public static Action<string, NamedElement> ValidationOnSchemaInitialize = NameValidator.ValidateDefault;
+
+        /// <summary>
+        /// This setting improves performance if your schema uses only scalar types or types marked with
```

Comment:
This only improves performance during schema build time, correct?  I think that is important to note here, as there is no practical reason to turn it off unless you have a scoped schema.  Can we update the note to indicate as such?

Reply:
Done